python/lsst/ts/scheduler/driver/feature_scheduler.py

```diff
@@ -74,6 +81,10 @@ class FeatureSchedulerParameters(DriverParameters):
 class FeatureScheduler(Driver):
     """Feature scheduler driver."""
 
+    default_observation_database_name = (
+        pathlib.Path.home() / "fbs_observation_database.sql"
+    )
+
     def __init__(self, models, raw_telemetry, parameters=None, log=None):
```

Comment:
Please add a type definition for the parameters and a return type for the method.